test/modelValidatorTests.ts

```diff
@@ -57,8 +56,14 @@ describe("Model Validation", () => {
         `swagger "${specPath} with operation "${operationIds}" should report only 1 error.`
       )
 
+      if (result[0].errorDetails === undefined) {
+        throw new Error("result[0].errorDetails === undefined")
+      }
+      if (result[0].errorDetails.similarPaths === undefined) {
```

Comment:
will similarPaths be undefiend if there are no similar paths or just an empty array ? would prefer empty array

Reply:
NodeError may not have `similarPaths`